packages/shared/src/networks/HemiMainnet.ts

```diff
@@ -0,0 +1,3 @@
+import { hemi } from 'hemi-viem'
+
+export const HemiMainnet = hemi
```

Comment:
@ArturDolzan  Do you think it makes sense to have an abstraction of the networks at the domain level? This would help avoid any dependency on `viem`.